inscriptions/ISic3068.xml

```diff
@@ -185,43 +190,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI332227">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — —</supplied>
-                    <lb n="2"/>ΑΚΕ․
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="3"/>Κάλλ<supplied reason="lost">ων</supplied>-<lb n="4"/>ος τόδ
-                    <supplied reason="lost">ε</supplied>
-                    <lb n="5"/>σῆμ<supplied reason="lost">α π</supplied>-<lb n="6"/>ερίφρ<supplied reason="lost">
-                    ον</supplied>-<lb n="7"/>ος ἤν
-                    <supplied reason="lost">υσε</supplied>
-                    <lb n="8"/>μ
-                    <unclear>ήτ</unclear>
-                    <supplied reason="lost">ηρ</supplied>.
-                    <lb n="9"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI336195">
-                <ab>
-                    <lb n="1"/>
-
-
-                    Ἀκε
-                    <unclear>σ</unclear>
-                    <supplied reason="lost">ία(?) τοῦ</supplied>
-                    <lb n="2"/>Καλλ<supplied reason="lost">ικράτε</supplied>-<lb n="3"/>ος τόδ
-                    <supplied reason="lost">ε καλὸν(?)</supplied>
-                    <lb n="4"/>σῆμ
-                    <supplied reason="lost">α — — —</supplied>
-                    <lb n="5"/>Ἐριφρ<supplied reason="lost">άδε</supplied>-<lb n="6"/>ος ἡ μ
-                    <supplied reason="lost">έδουσα(?)</supplied>
-                    <lb n="7"/>μήτ<supplied reason="lost">ηρ ἐποίει</supplied>.
-                    <lb n="8"/>
+                    <lb n="0"/><gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="1"/>ΑΚΕ<gap reason="illegible" quantity="1" unit="character"/><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/>Κάλλ<supplied reason="lost">ων</supplied>
+                    <lb n="3" break="no"/>ος τόδ<supplied reason="lost">ε</supplied>
+                    <lb n="4"/>σῆμ<supplied reason="lost">α</supplied> <supplied reason="lost">π</supplied>
+                    <lb n="5" break="no"/>ερίφρ<supplied reason="lost">ον</supplied>
+                    <lb n="6" break="no"/>ος ἤν<supplied reason="lost">υσε</supplied>
+                    <lb n="7"/>μ<unclear>ήτ</unclear><supplied reason="lost">ηρ</supplied>
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after Manganaro 1999</note> <!-- ?check: available only Orsi 1897 -->
```

Comment:
I have Manganaro 1999 and must scan it; but I also have autopsy and photographs of this one. But can be followed up in due course, leave as is for now.